e2e/terraform/config/shared/nomad-tls.hcl

```diff
@@ -18,3 +18,12 @@ consul {
   cert_file = "/etc/nomad.d/tls/agent.crt"
   key_file  = "/etc/nomad.d/tls/agent.key"
 }
+
+vault {
+  enabled   = true
+  address   = "https://active.vault.service.consul:8200"
+
+  ca_file   = "/etc/nomad.d/tls/ca.crt"
+  cert_file = "/etc/nomad.d/tls/agent.crt"
+  key_file  = "/etc/nomad.d/tls/agent.key"
+}
```

Comment:
I recommend that you configure your editor to ensure files always have EOF (trailing newlines) on save. It's safe here because Nomad [doesn't blindly stitch the text together](https://github.com/hashicorp/nomad/blob/v1.2.2/command/agent/config.go#L1969-L1982), but it's a good practice in general. Because it's [part of the posix standard for "text" files](https://unix.stackexchange.com/questions/446237/what-conditions-must-be-met-for-a-file-to-be-a-text-file-as-defined-by-posix), lots of the tools you use on a daily basis will expect it and it can result in unexpected behavior. A quick example, but you'll also see this kind of thing with `grep`:

```sh
$ cat missingnewline.hcl
vault {
  enabled   = true
  address   = "https://active.vault.service.consul:8200"

  ca_file   = "/etc/nomad.d/tls/ca.crt"
  cert_file = "/etc/nomad.d/tls/agent.crt"
  key_file  = "/etc/nomad.d/tls/agent.key"
}%
$ cat hasnewline.hcl
vault {
  enabled   = true
  address   = "https://active.vault.service.consul:8200"

  ca_file   = "/etc/nomad.d/tls/ca.crt"
  cert_file = "/etc/nomad.d/tls/agent.crt"
  key_file  = "/etc/nomad.d/tls/agent.key"
}

$ diff missingnewline.hcl hasnewline.hcl
8c8
< }
\ No newline at end of file
---
> }
$ wc -l hasnewline.hcl
       7 hasnewline.hcl
$ wc -l hasnewline.hcl
       8 hasnewline.hcl
```

Reply:
Thanks!  Will do.

Reply:
Done.